packages/openid4vc-client/src/OpenId4VcClientApi.ts

```diff
@@ -23,13 +22,29 @@ export class OpenId4VcClientApi {
     this.openId4VcClientService = openId4VcClientService
   }
 
-  public async requestCredentialPreAuthorized(options: PreAuthorizedOptions): Promise<W3cCredentialRecord> {
+  public async requestCredentialUsingPreAuthorizedCode(options: PreAuthCodeFlowOptions): Promise<W3cCredentialRecord> {
+    // set defaults
+    const checkRevocationState = options.checkRevocationState ?? true
```

Comment:
Is there a reason it's called checkRevocationState here and verifyRevocationState in the w3c service?